src/rise-twitter.js

```diff
@@ -64,8 +64,12 @@ export default class RiseTwitter extends HTMLElement {
       });
 
       risePlaylistItem.addEventListener('play', () => {
-        this._play();
-        this.logger.playlistEvent('Play Event');
+        if (this.settings.getIsAuthorized()) {
```

Comment:
@jungeunyoon I think this code could be add to the play method. I am just calling the play method within a arrow function here so the "this" gets passed to the _play function scope. Thoughts? Cheers

Reply:
@rodrigopavezi I had it that way originally but then realized that `_play` would break the [https://docs.google.com/document/d/1eQRTDb7bK2BvCMEec0AxZHE7QHbIim3-KMbukgXEegI/edit#bookmark=id.tlko7617ian3](single responsibility principle) and that we wouldn't be able to test authorization cases because of it. 

let me reorganize this a bit. it's a valid point. thanks!